content/working_with/official_plugins/azure.md

```diff
@@ -37,6 +37,8 @@ The SDK-based method is dependent on the SDK library versions. (See the setup.py
 
 Each Azure resource node template must include a property `azure_config` for authentication. This consists of a `tenant_id`, `client_id`, `client_secret` and `subscription_id`. These can be provided via secrets for better security coverage.
```

Comment:
@EarthmanT maybe we need to update this (line 38) as now it accept either `client_secret` or `client_assertion`